lib/protoparser/datadog/api/sketches/beta/proto/sketches.proto

```diff
@@ -0,0 +1,112 @@
+syntax = "proto3";
```

Comment:
would be rly great if you add links/docs for:
1. How to find sketches proto in upstream
2. what modification was made to this proto and why
3. how to generate .pb.go from it (like how to maintain it in future)